indico/modules/logs/controllers.py

```diff
@@ -245,6 +247,25 @@ def _build_email(self, form_data, is_html_email):
                 body = f'{preface}{text_separator}{body}'
                 alternatives = []
 
+        # Rebuild attachments from storage if available
+        stored_atts = email_data.get('stored_attachments') or []
+        if stored_atts:
+            for item in stored_atts:
+                try:
+                    storage = get_storage(item['storage_backend'])
+                    with storage.open(item['storage_file_id']) as fd:
+                        content = fd.read()
+                except StorageError:
+                    # Do not fail resending if an attachment cannot be retrieved
+                    continue
+                filename = item.get('filename') or 'attachment'
+                ctype = item.get('content_type')
+                if ctype:
```

Comment:
I don't think this check is needed, we always have a content type, so you can always use the 3-tuple-version.